### DISCUSSION TITLE
output the simulation result as a gif/video

### DISCUSSION THREAD

**Discussion by BoyuanTang331** [InteractiveComputerGraphics/SPlisHSPlasH#301] [Q&A] [answered]
Hi,

is there any easy way to print out the pySplishSplash result as a gif or video?

**Comment by janbender** [comment, marked as answer]
SPlisHSPlasH can export the data as bgeo files and the tool PartioViewer (https://splishsplash.readthedocs.io/en/latest/PartioViewer.html) can load the data and generate a video (using ffmpeg). Alternatively, you can generate a video easily using ParaView.
